src/NodeJS/Jering.Javascript.NodeJS.csproj

```diff
@@ -52,11 +51,11 @@
         <PackageReference Include="Yarn.MSBuild" Version="1.16.0" PrivateAssets="all" ExcludeAssets="Runtime">
             <IncludeAssets>compile; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-        <PackageReference Include="System.Buffers" Version="4.5.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.2.0" />
-        <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0" />
-        <PackageReference Include="Microsoft.Extensions.Options" Version="3.0.0" />
+        <PackageReference Include="System.Text.Json" Version="4.6.0" />
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.0" />
```

Comment:
This looks excellent, seems like it makes the project compatible with an AspNetCore project targeting dotnetcore2.1 while still supporting dotnetcore3.0. Is that the reason for downgrading?

Reply:
Yes, because net core 2.1 and aspnetcore 2.1 is LTS :)